.azure-pipelines/win32-conda-CI.yml

```diff
@@ -19,13 +19,11 @@ jobs:
       Python35:
         python.version: '3.5'
         ONNX_PATH: onnx==1.2.3
-        KERAS: keras==2.1.6
         COREML_PATH: https://github.com/apple/coremltools/archive/v2.0.zip
 
       Python36:
-        python.version: '3.6'
+        python.version: '3.6.7'
```

Comment:
which converter is specific to this python version?

Reply:
I wanted to check if the issue with numpy could be solved without a previous python version. I'll remove it.